CMakeLists.txt

```diff
@@ -179,6 +183,20 @@ add_thirdparty_package(
   DEPENDS ${PYTHON3_TARGET} ${LIBICONV_TARGET}
 )
 
+add_thirdparty_package(
```

Comment:
If we are packaging prjxray into conda, why is this required?

Reply:
This is for the python dependencies for the toolchain_wrappers

Reply:
But if prjxray is installed from conda, why isn't the python library already instead?

Reply:
Currently the conda-package contains only the binary tools, and not the python library.
We could add python libraries as well in the conda package to avoid installing the third_party package here

Reply:
I think the current naming scheme is confusing.  prjxray is made up of multiple parts.  The bitstream tools, the python library, the fuzzers and the database output.  If this package only has the tools, it should be probably be called prjxray-tools or xc7bittools, etc.

Reply:
Ok, than, it would be ok to leave this thirdparty package and modify the conda-package naming, correct?